Public/Src/Utilities/KeyValueStore/RocksDb/RocksDbStore.cs

```diff
@@ -197,6 +197,15 @@ private struct DefaultOptions
                     .SetBlockBasedTableFactory(blockBasedTableOptions)
                     .SetPrefixExtractor(SliceTransform.CreateNoOp());
 
+                // Maximum number of information log files
+                m_defaults.DbOptions.SetKeepLogFileNum(1000);
+
+                // Do not rotate information logs based on file size
+                m_defaults.DbOptions.SetMaxLogFileSize(0);
+
+                // How long before we rotate the current information log file
+                m_defaults.DbOptions.SetLogFileTimeToRoll((ulong)TimeSpan.FromHours(1).Seconds);

```

Comment:
Should we rotate every day instead?
Rotating every hour seems a bit too frequent?

Reply:
Rotating every 12 hours to avoid loosing an entire day out as a result of master changing.